dt-metrics/contacts/seeker-path.php

```diff
@@ -61,7 +61,7 @@ public function scripts() {
                 'translations' => [
                     'seeker_path' => __( "Seeker Path", 'disciple_tools' ),
                     'filter_contacts_to_date_range' => __( "Filter contacts to date range:", 'disciple_tools' ),
-                    'all_time' => __( "All time", 'disciple_tools' ),
+                    'all_time' => _x( "All Time", 'Dates', 'disciple_tools' ),
```

Comment:
Just have them be all without context. All time is a string used in many places, so we are bound to have duplicates

Reply:
I will adjust this to be without the context, but have the text be the same as other places which is 'All Time'.